operator/standalone/controller.go

```diff
@@ -15,7 +15,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-const finalizer = "finalizer"
+var finalizer = strings.ReplaceAll(v1alpha1.Group, ".", "-")
+
+var (
+	// OperatorNamespace is the namespace where the controller looks for v1alpha1.PostgresqlStandaloneOperatorConfig.
+	OperatorNamespace = ""
+	// ServiceNamespacePrefix is the namespace prefix which the controller uses to create the namespaces where the PostgreSQL instances are actually deployed in.
+	ServiceNamespacePrefix = "sv-postgresql-"
```

Comment:
Similar here. If we keep that general prefix also for replicated or cluster, it could happen, that a customer can provision a standalone and cluster in the same target namespace.

Reply:
That's true, I haven't thought of that.
However, it depends how we actually want to name the namespace. Right now, I went for <prefix>-<namespace>-<instance>, but that heavily limits the amount of characters the user can choose, since the namespace cannot exceed a total of 63 characters.
Choosing a generative name like `sv-postgresql-afroiu9023f4n` should avoid all collisions by simply reducing the probility, but the UX is not great... Do you have other ideas to manage this problem?

Reply:
Maybe we could shorten the prefix then?

Something like: `sv-psql-s-`

Where s indicates that this is a standalone instance. This leaves more characters for the customer to choose.

I'd also go for a fixed length random string instead of adding the namespace. Because the namespace itself could already be close to the 63 char limit. If the source namespace is of any relevance to us, we should add it as an annotation or label.

Given the character limit and no randomization in the name, it's probably always possible to create clashes with carefully chosen names.

Reply:
Maybe some halfway-recognizable haikunated name:
`sv-postgresql-standalone-wild-flower-3456`?

There could multiple instances with same name, differentiated only by namespace, e.g. `drupal-dev/database` and `drupal-prod/database`...

What I probably would avoid, is having 2 naming schemes, e.g. `<someprefix>-<namespace>-<instance>` if the name fits, and using alphanumeric generated names if the first attempt exceeds 63 chars. That creates a confusing inconsistency.

Reply:
> What I probably would avoid, is having 2 naming schemes, e.g. <someprefix>-<namespace>-<instance> if the name fits, and using alphanumeric generated names if the first attempt exceeds 63 chars. That creates a confusing inconsistency.

I agree.

> Maybe some halfway-recognizable haikunated name:

Sounds good to me.